docs/editor_manual/administrator_tasks/managing_users.md

```diff
@@ -0,0 +1,34 @@
+# Managing users and roles
+
+As an administrator, a common task will be adding, modifying or removing user profiles.
+
+This is done via the 'Users' interface, which can be found in the Settings menu, accessible via the left-hand menu bar.
+
+```{image} ../../_static/images/screen35_users_menu_item.png
+```
+
+In this interface you can see all of your users, their usernames, their 'level' of access (otherwise known as their 'role'), and their status, either active or inactive.
+
+You can sort this listing either via Name or Username.
+
+```{image} ../../_static/images/screen36_users_interface.png
+```
+
+- Select multiple users by checking the checkbox on the left of each row, then use the bulk actions bar at the bottom to perform an action on all selected users.
+
+```{image} ../../_static/images/screen36.5_users_bulk_actions.png
+```
+
+Clicking on a user's name will open their profile details. From here you can then edit that users details.
+
+:::{Note}
+It is possible to change user's passwords in this interface, but it is worth encouraging your users to use the 'Forgotten password' link on the login screen instead. This should save you some time!
+:::
```

Comment:
I know that this format for admonitions will work, however we have documented the backticks approach as the preferred option.

https://docs.wagtail.org/en/latest/contributing/documentation_guidelines.html#note-and-warning-call-outs

Please can this, and other notes (admonitions) use the backtick syntax as follows.

```suggestion
```{note}
It is possible to change user's passwords in this interface, but it is worth encouraging your users to use the 'Forgotten password' link on the login screen instead. This should save you some time!
```
```